run/running.example.ini

```diff
@@ -1,11 +1,9 @@
 [default]
 build_path = $HOME/wptdbuild
-wpt_path = $HOME/gh/w3c/web-platform-tests
-wptd_path = $HOME/gh/w3c/wptdashboard
+wpt_path = $HOME/web-platform-tests
+wptd_path = $HOME/wptdashboard
```

Comment:
This change is unrelated. I always have to change this when setting up a bot. Sneaking in a change to the default here.